### PR TITLE
merge of code and clean up of repo

### PR DESCRIPTION
**Pull request by Brownr151**

### CODE REVIEW

File: terraform.tf [hunk 1]
```diff
@@ -21,9 +21,9 @@ variable "aws_region" {
 module "lambda_function" {
   source = "terraform-aws-modules/lambda/aws"
 
-  function_name = "my-lambda"
-  description   = "My awesome lambda function"
-  handler       = "lambda_code/hello_world.lambda_handler"
+  function_name = "new-lambda"
```
Comment (StephenThomson) [inline terraform.tf:24]: Describe what the component actually does.

File: terraform.tf [hunk 2]
```diff
@@ -21,9 +21,9 @@ variable "aws_region" {
...
+  description   = "My updated lambda function"
```
Comment (StephenThomson) [inline terraform.tf:25]: Same here

File: terraform.tf [hunk 3]
```diff
@@ -32,7 +32,7 @@ module "lambda_function" {
   # source_path = "../InstanceProjGroupWork/hello_world.py"
 
   tags = {
-    Name = "my-lambda"
+    Name = "new-lambda"
```
Comment (StephenThomson) [inline terraform.tf:35]: As Above

**Review by StephenThomson — CHANGES_REQUESTED** [review]
Couple of comments and please don't push up the zip file. Maybe add to git ignore?

**Comment by StephenThomson** [comment]
Also please don't push up all the python dependencies

File: terraform.tf (hunk 3 above)
Comment (Masquerainsley) [reply terraform.tf]: Hey steve, can you explain why we shouldn't push up the zip file?

**Review by AshleyDD1992 — COMMENTED** [review]
More descriptive names and comments

File: terraform.tf (hunk 2 above)
Comment (AshleyDD1992) [reply terraform.tf]: updated

File: terraform.tf (hunk 3 above)
Comment (AshleyDD1992) [reply terraform.tf]: updated

Comment (StephenThomson) [reply terraform.tf]: 1. It's big and we don't need it.
2. There is a risk it goes out of date.
3. We can just make it again with our build process.
4. There is a chance we deploy old code if we don't build properly.

File: terraform.tf (hunk 1 above)
Comment (AshleyDD1992) [reply terraform.tf]: updated

File: terraform.tf (hunk 3 above)
Comment (Masquerainsley) [reply terraform.tf]: cheers :)

**Pull request merged**